### PR TITLE
DOC-3681: Update location of AWS DMS doc in sidebar

### PR DESCRIPTION
**Pull request by nickvigilante**
Fixes DOC-3681

Also adds references to AWS DMS inside existing DB migration docs

### CODE REVIEW

**Review by jseldess — CHANGES_REQUESTED** [review]
LGTM overall, but I think the title of the dms doc page is still misleading, as it focuses on schemas?
`Migrate CockroachDB Schemas with AWS Database Migration Service (DMS)`

**Comment by nickvigilante** [comment]
@jseldess Oops, I updated the links to that page but not the page title itself. Check again?

File: v22.1/aws-dms.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
-title: Migrate CockroachDB Schemas with AWS Database Migration Service (DMS)
+title: Migrate CockroachDB Data with AWS Database Migration Service (DMS)
```
Comment (jseldess) [inline v22.1/aws-dms.md:2]: I'm still a little unsure about this. We're not migrated cockroachdb data. We're migrating data to cockroachdb.

What about just `Migrate with AWS Database Migration Service (DMS)`?

Comment (nickvigilante) [reply v22.1/aws-dms.md]: I like that better. Thanks for the suggestion!

**Review by jseldess — APPROVED** [review]
LGTM

**Review by awoods187 — APPROVED** [review]
> Reviewed 1 of 4 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14028)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ajstorm and @jseldess)

**Pull request merged**